UI/window-basic-preview.cpp

```diff
@@ -49,6 +49,18 @@ vec2 OBSBasicPreview::GetMouseEventPos(QMouseEvent *event)
 
 	return pos;
 }
+vec2 OBSBasicPreview::GetWheelEventPos(QWheelEvent *event)
+{
+	OBSBasic *main = reinterpret_cast<OBSBasic *>(App()->GetMainWindow());
+	float pixelRatio = main->GetDevicePixelRatio();
```

Comment:
Please check if this function is only called in the main thread - if it is called from the GPU thread, you need to use `OBSBasic::GetDevicePixelRatio()` instead. Calling UI functions off the UI thread is no bueno.

Reply:
The only time I call this function is in response to a mousewheel event, so I would expect that it should only be called on the UI thread -- this is a direct copy of a similar function that just uses a different event type. I would assume if the other one is safe then this one should be safe as well. Do you have any reason to expect otherwise?